### ISSUE TITLE
run on win10

### ISSUE THREAD

**Issue by wdy0311** [JiehongLin/SAM-6D#80]
may I ask how to run sam-6d project on win10? it seems that the installation is based on linux.

**Comment by SajjadPSavoji** [comment]
try using the docker image they provide

**Comment by wdy0311** [comment]
> try using the docker image they provide

thx, could you plz do me a favor and tell me the way how to use it? it seems there is no project files in the docker but the environment is ok

**Comment by SajjadPSavoji** [comment]
> > try using the docker image they provide
> 
> thx, could you plz do me a favor and tell me the way how to use it? it seems there is no project files in the docker but the environment is ok

right! There might be several ways to go about this. What I personally do is I clone the repo in my local and mount the repo to /workspace in the docker container.

https://www.geeksforgeeks.org/mounting-a-volume-inside-docker-container/

Hope this helps
